Library/Homebrew/os/mac/mach.rb

```diff
@@ -51,8 +51,10 @@ def mach_data
     end
   end
 
-  def dynamically_linked_libraries
-    macho.linked_dylibs
+  def dynamically_linked_libraries(except: [])
+    lcs = macho.dylib_load_commands.reject { |lc| [*except].include?(lc.type) }
```

Comment:
Looks like you're expecting `expect` to be an array here but it's a non-array above. I'm happy with either but would be good to be consistent. Additionally, if you're expecting an array I think `expect` is equivalent to `[*except]`, no?